AdminApi/src/AdminApi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.ts

```diff
@@ -93,12 +93,29 @@ export class IdentityDetailsComponent {
         this.tierService.getTiers().subscribe({
             next: (tiers) => {
                 this.tiers = tiers.result;
+                if (this.identity.tierId !== "TIR00000000000000001") {
+                    this.tiers = this.tiers.filter((tier) => tier.id !== this.getQfDTierId());
+                }
                 this.updatedTier = this.tiers.find((t) => t.id === this.identity.tierId);
                 this.tier = this.updatedTier;
             }
         });
     }
 
+    public enablesManualAssignment(tier: TierOverview): boolean {
+        return tier.canBeManuallyAssigned;
+    }
+
+    public disabledTiers(tier: TierOverview): boolean {
+        const isQfDIdentityTierId = this.identity.tierId === this.getQfDTierId();
```

Comment:
Why do you even check for the id of the QueuedForDeletionTier? The purpose of the properties we have introduced is to prevent the need to check for any ids.

Reply:
Yes i am aware of that, but still i am not sure how to check if the Queued For Deletion tier is set in identity it self, I mean if it is set to an identity then i need some check to verifiy it and thats how i am disableing all other tiers except Queued For Deletion tier.